.github/workflows/ci.yml

```
@@ -241,6 +239,7 @@ jobs:
         uses: docker/setup-buildx-action@6524bf65af31da8d45b59e8c27de4bd072b392f5 # v3.8.0
 
       - name: Log in to the container registry
+        if: env.PUSH_IMAGE == 'true'
```

```suggestion
        if: env.PUSH_IMAGE
```

Sadly, this doesn't work.